### ISSUE TITLE
incorrect id received when using stream loading

### ISSUE THREAD

**Issue by pallavg** [Toblerity/rtree#50]
Using rtree 0.8.2. The small example below recreates the problem.

#!/usr/bin/env python

from rtree import index

def gen(d):
   for i,obj in enumerate(d):
      yield (i, obj[0], obj[1])

data = [((0,0,1,1), 25), 
        ((1,1,2,2), 30),
        ((2,2,3,3), 35)]

idx = index.Index('foo', gen(data))

m = idx.intersection((2,3,3,3), objects=True)
for a in m:
        print a.id
        print a.bbox
        print a.object

It correctly prints the correct bbox/object but the wrong id (should be 2). It seems like when stream loading is used to create the index, the ID is always returned as 0. It seems to work OK if idx.insert() is used to create the tree. Thanks.
                                                                                                                                             
$ ./test.py 
0
[2.0, 2.0, 3.0, 3.0]
35

**Comment by mthh** [comment]
I'm not sure if it is an issue (or maybe the documentation is unclear about the `object='raw'` parameter), but I guess i had the same result (only getting 0 returned after using the stream loading method).
I solved it by passing the id in the field dedicated for the object :

```python
def idx_generator_function(bounds):
    for i, bound in enumerate(bounds):
        yield (i, bound, i)
# Note the last i (and also the fact that i wasn't interested in having the object returned),
# (+ that the first i is probably useless when the function is used like this)

def make_index(bounds):
    return rtree.index.Index([z for z in idx_generator_function(bounds)],
                             Interleaved=True)
```

Then looking for the ids and objects by doing :
```python
index = make_index([i.bounds for i in geomsA])
for geom in geoms:
    for fid in list(index.intersection(geom.bounds, objects='raw')):
        # ..do something with geomsA[fid] 
```

**Comment by hobu** [comment]
Fixed in #77
